.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+| Question         | Answer
+| ---------------- | ---
+| Branch?          | `master` for hotfix / `develop` for bug fixes <!-- see below -->
+| Bug fix?         | yes/no
+| New feature?     | yes/no
+| Breaking change? | yes/no
+| Tickets          | Fix #... <!-- prefix each issue number with "Fix #", if any -->
+| License          | BSD 3-Clause
```

Comment:
I think about:

```
| I have read the rules before submitting this PR          | [ ]
```

Reply:
That's the purpose of the CONTRIBUTING.md file I would say.

Reply:
`CONTRIBUTING.md` states the rules, the rules states that you have to read them, but the user never says he actually read them before this step 😉

Reply:
Users are invited to read them when they contributes:

https://help.github.com/en/github/building-a-strong-community/setting-guidelines-for-repository-contributors

Reply:
And they never tell us that they did :)